src/compile_commands.cc

```diff
@@ -0,0 +1,79 @@
+#include "compile_commands.h"
+#include <boost/property_tree/json_parser.hpp>
+
+std::vector<std::string> CompileCommands::Command::paramter_values(const std::string &paramter_name) const {
+  std::vector<std::string> parameter_values;
+  
+  bool found_argument=false;
+  for(auto &parameter: parameters) {
+    if(found_argument) {
+      parameter_values.emplace_back(parameter);
+      found_argument=false;
+    }
+    else if(parameter==paramter_name)
+      found_argument=true;
+  }
+  
```

Comment:
Ohh, sorry I missed the else if, but at least this made me notice the miss spelt `paramter_name`.